src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/reset_global_properties_request.cc

```diff
@@ -197,12 +208,18 @@ bool ResetGlobalPropertiesRequest::PrepareResponseParameters(
   } else {
     result =
         PrepareResultForMobileResponse(ui_properties_info, tts_properties_info);
+    result = (result || PrepareResultForMobileResponse(ui_properties_info,
+                                                       rc_properties_info));
     out_result_code =
         PrepareResultCodeForResponse(ui_properties_info, tts_properties_info);
-    out_response_info = app_mngr::commands::MergeInfos(tts_properties_info,
-                                                       tts_response_info_,
-                                                       ui_properties_info,
-                                                       ui_response_info_);
+    if (rc_result_ != hmi_apis::Common_Result::INVALID_ENUM) {
+      out_result_code = std::max(
```

Comment:
@RomanReznichenkoLuxoft same as above - please create overload for `PrepareResultCodeForResponse` in the base class which accepts three arguments and use it (note that `MergeInfos()` already has overloaded function which accepts three arguments)

Reply:
Done